documentation/asciidoc/computers/configuration/configuring-networking.adoc

```diff
@@ -86,7 +86,7 @@ On the Raspberry Pi 3B+ and Raspberry Pi 4B, you will also need to set the count
 country=GB
 ----
 
-Note that with the latest Buster Raspberry Pi OS release, you must ensure that the `wpa_supplicant.conf` file contains the following information at the top:
+Note that with the latest Raspberry Pi OS release, you must ensure that the `wpa_supplicant.conf` file contains the following information at the top:
```

Comment:
@aallan Do you know if `wpa_supplicant.conf` is still used in Bookworm? That's something I've not tested/investigated yet.

Reply:
This suggests that it does, must admit I configure my WiFi in Imager these days so havent' checked. https://manpages.debian.org/bookworm/wpasupplicant/wpa_supplicant.conf.5.en.html

Reply:
Well let's leave it in for now then, and we can always edit it out as needed as part of #2956